src/main/resources/ValidationMessages.properties

```diff
@@ -1,2 +1,3 @@
-company.number.NotEmpty.message=You must enter a company number
-company.number.Size.message=You must enter a valid company number
\ No newline at end of file
+company.number.NotEmpty.message=Enter the company number
+company.number.Size.message=If it''s 7 characters or less, enter zeros at the start so that it''s 7 characters in total.
```

Comment:
Yes, the validation message is shown when the length is not 8, as per the acceptance criteria

Acceptance Criteria 4
Given that the user enters a company number which contains less or more than 8 characters 
When they click Continue
Then the following error message is shown

If it’s 7 characters or less, enter zeros at the start so that it's 8 characters in total.